tests/Module/Checkout/SessionServiceTest.php

```diff
@@ -49,6 +49,21 @@ public function testGetChosenMethodFromSessionWithNoShippingMethods(): void {
 		$this->assertEquals( '', $this->sessionService->getChosenMethodFromSession() );
 	}
 
+	public function testGetChosenMethodFromSessionWithoutShippingMethods(): void {
```

Comment:
Pojmenoval bych spíš `testGetChosenMethodFromSessionWithFalseShippingMethod`, takto je to zavádějící duplicita k `testGetChosenMethodFromSessionWithNoShippingMethods`